index.html

```diff
@@ -16,298 +16,390 @@
+                    See Project
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div class="card-flex">
+              <div class="description">
+                <h2 class="new">Multi-Post Stories Gain+Glory</h2>
+                <ul class="list p-lang">
+                  <li>
+                    <button class="btn-btn" type="button">Ruby on rails</button>
+                  </li>
+                  <li><button class="btn-btn" type="button">Css</button></li>
+                  <li>
+                    <button class="btn-btn" type="button">Javascript</button>
+                  </li>
+                  <li><button class="btn-btn" type="button">Html</button></li>
+                </ul>
+                <div class="sec2-btn">
+                  <button class="screen-xl btn view" type="button">
+                    See Project
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div class="card-flex">
+              <div class="description">
+                <h2 class="new">Multi-Post Stories Gain+Glory</h2>
+                <ul class="list p-lang">
+                  <li>
+                    <button class="btn-btn" type="button">Ruby on rails</button>
+                  </li>
+                  <li><button class="btn-btn" type="button">Css</button></li>
+                  <li>
+                    <button class="btn-btn" type="button">Javascript</button>
+                  </li>
+                  <li><button class="btn-btn" type="button">Html</button></li>
+                </ul>
+                <div class="sec2-btn">
+                  <button class="screen-xl btn view" type="button">
+                    See Project
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
       </div>
     </section>
 
-    <!-- recent works section -->
-    <section id="recent-works">
-      <div class="section-tag">
-        <h1 class="section-title">My Recent Works</h1>
-        <a><img src="images/Indicator.svg" alt="Indicator" /></a>
+    <!-- recent About-me section -->
+    <section id="about-me">
+      <div class="about">
+        <div class="about-li">
+          <img class="mob-img" src="images/Frame-1.svg" alt="side-image" />
+          <img
+            class="side2 lap-img"
+            src="images/illustration-about-me-2-desktop.svg"
+            alt="side-image"
+          />
+        </div>
+
+        <div class="about-li">
+          <h1 class="ab-head">About me</h1>
+          <p class="ab-text">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <button class="btn ab-btn-btn resume" type="button">
```

Comment:
:red_circle: [OPTIONAL] A close look at the Figma design shows that this button is centered. Kindly wrap it in a div container and set text-align:center to the  container.
You can do this :arrow_down: 
```
<!-- in index.html -->
<div class="text-center">
        <button class="btn ab-btn-btn resume" type="button">
</div>
// In style.css
.text-center {
    text-align: center;
}
```
Your Design
![ab-sec](https://user-images.githubusercontent.com/26736582/200121104-b87ae947-9263-46a2-b14c-d5ce70d61ba0.png)

Figma Design
![about-me](https://user-images.githubusercontent.com/26736582/200121227-1d12f437-0c63-4a93-9430-42e36d3ce3f3.png)